pureStorageDriver/README.md

```diff
@@ -169,6 +169,8 @@ volumesnapshotclasses.snapshot.storage.k8s.io    2019-11-21T17:25:23Z
 volumesnapshotcontents.snapshot.storage.k8s.io   2019-11-21T17:25:23Z
 volumesnapshots.snapshot.storage.k8s.io          2019-11-21T17:25:23Z
 ```
+Note that PSO CSI has upgraded its sidecars to support BETA version snapshotter APIs. The snapshotter CRDs for Beta version APIs have been upgraded, make sure for 1.17 and later versions, apply release-2.0 CRDs as shown
```

Comment:
I think this whole added section needs to be more prominent - it is crucial that users are aware of this to stop unnecessary escallations

Reply:
will do